Finish mobile version

LarryIVC opened the pull request:
### **In this PR, I did the following:**

- Create a new branch to develop the Works and About Myself sections.
- Added HTML layout for Works and About Myself sections.
- Added Styles to the Works and About Myself sections.
- Test project whit linter.

LarryIVC committed:
Adding images to image dir

LarryIVC committed:
Changing layout of Works,About sections on styles.css

LarryIVC committed:
Fix stylelint errors on styles.css

LarryIVC committed:
Clear comments on styles.css

leonardodiasb commented on index.html:
```diff
@@ -25,21 +25,178 @@
+            <div class="about-skills">
+                <ul>
+                    <li>
+                        <div class="languages spc-36">
+                            <span>Languages</span><img src="image/down-ico.png" alt="Arrow down">                            
+                        </div>
+                    </li>
+                        <div class="lng-item">
+                            <img src="image/js-logo.png" alt="JS logo"><span>JavaScript</span>
+                        </div>
+                        <div class="lng-item">
+                            <img src="image/html-logo.png" alt="HTML logo"><span>HTML</span>
+                        </div>
+                        <div class="lng-item">
+                            <img src="image/css-logo.png" alt="CSS logo"><span>CSS</span>
+                        </div>
```
- [x] The list of languages works like a list and they would better fit inside the `li` of the languages. Every item inside a `ul` should be wrapped with the `li` tag for better semantics so please move the 'javascript', 'html' and 'css' items to inside the `li` of the languages above it.

LarryIVC replied:
I move the 'JavaScript', 'HTML' and 'CSS' items to inside the li of the languages above it.
![image](https://user-images.githubusercontent.com/110146573/221306719-7e56c442-bb95-473f-9a1b-142590f287cd.png)

leonardodiasb commented on styles.css:
```diff
@@ -60,21 +64,221 @@ main {
   margin-right: 20px;
 }
+
+.works {
+  width: 100%;
+  margin-top: 114px;
+  padding: 0 16px;
+}
```
- [x] As the project required, it is mandatory to use flexbox and grid. For the main container of the works section, you should use css grid. Kindly use it to match the project requirements.

LarryIVC replied:
@leonardodiasb, I insert `css grid` in the works section.
![image](https://user-images.githubusercontent.com/110146573/221319123-af30883f-c9b7-4579-94d8-c90fd3ea759b.png)

leonardodiasb commented on styles.css:
```diff
@@ -60,21 +64,221 @@ main {
+  padding: 0 16px;
+}
+
+.works1 {
+  width: 100%;
+  background: white;
+  border-radius: 16px;
+  padding: 16px;
+  margin-bottom: 88px;
+  border: #dfe1e6 solid 1px;
+}
```
- [x] As the project required, it is mandatory to use flexbox and grid. For each individual project of the works section, you should use CSS flexbox. Kindly use it to match the project requirements.

LarryIVC replied:
@leonardodiasb 
I adjust `Flexbox` in sections, Obs 3
![image](https://user-images.githubusercontent.com/110146573/221338200-efc62eaa-127a-4377-b332-3d335faa5fb6.png)

leonardodiasb reviewed (changes requested):
# Required Changes ♻️ ⚠️ 🚦 

Hi @LarryIVC, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

![mario_ladders](https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif)

## Good points 👍 

 - Very professional README and PR message. ✔️
 - No linter errors. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

LarryIVC committed:
Fit inside the li Obs1

LarryIVC committed:
Insert Grid in Works Obs 2

Observation 2 from PR7 review 1

LarryIVC committed:
Set the Flexbox CSS

Observation 3
Insert Flexbox into the project
Section:
  - main
  - pg1-container
  - social
  - works1
  - small-button
  - sub-box
  - about-desc
  - lang
  - languages
  - lng-item

divinecharlotte reviewed (approved):
# Hello @LarryIVC   😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 

<img src="https://media3.giphy.com/media/3KC2jD2QcBOSc/giphy.gif"/>

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
